NuGet.config

```diff
@@ -5,11 +5,10 @@
   </solution>
   <packageSources>
     <clear />
-    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
Does `nuget.org` was causing a problem? or it is not needed anymore?

Reply:
Resolved per offline convo.